src/main/resources/reference.conf

```diff
@@ -803,22 +814,30 @@ resourceTypes = {
       modify = {
         description = "modify attributes of the cluster"
       }
+      set_parent = {
+        description = "set parent of notebook cluster"
+      }
     }
     ownerRoleName = "creator"
     roles = {
       creator = {
         roleActions = ["status", "connect", "delete", "read_policies", "stop_start", "modify"]
       }
+      manager = {
+        roleActions = ["status", "delete", "read_policies"]
+      }
     }
     reuseIds = false
+    authDomainConstrainable = true
```

Comment:
I blindly followed this pattern from other resources -- does `authDomainConstrainable = true` need to be specified here at the resource level, in addition to the action level?

Reply:
Interesting… I'll just set it at the action level then I think.